src/lib.rs.in

```diff
@@ -31,13 +31,13 @@ class PrivateKey {
 });
 
 foreigner_class!(
-/// Signing keypair specific to a device. Used to sign all requests to the IronCore API
+/// Signing private key specific to a device. Used to sign all requests to the IronCore API
 /// endpoints. Needed to create a `DeviceContext`.
-class DeviceSigningKeyPair {
-    self_type DeviceSigningKeyPair;
+class DeviceSigningPrivateKey {
```

Comment:
I'm not convinced this is a good rename.

Reply:
I undid this change. We'll want to change it back in ironoxide-scala as well at some point